pkgs/by-name/en/endgame-singularity/package.nix

```diff
@@ -44,6 +45,15 @@ python3.pkgs.buildPythonApplication {
           # ↑ we cannot glob on [...]/music, it doesn't exist yet
   '';
 
+  passthru = {
+    # for the updater
+    src = main_src;
+    updateScript = unstableGitUpdater {
+      tagPrefix = "v";
+      url = "https://github.com/singularity/singularity.git";
```

Comment:
you set `passthru.src`, so `url` is redundant

Reply:
Oh, that's handy! Dropped the `url`.